### PR TITLE
Convert to vanilla js

### PR DESCRIPTION
**Pull request by sasharevzin**
By using http://youmightnotneedjquery.com/ website it was very easy to convert it 💯

### CODE REVIEW

File: app/views/render_async/_render_async.html.erb [hunk 1]
```diff
@@ -4,10 +4,21 @@
 
 <% content_for :render_async do %>
   <%= javascript_tag html_options do %>
-    (function($){
-      $.ajax({ url: "<%= path %>" }).always(function(response) {
-        $("#<%= container_id %>").replaceWith(response);
-      });
-    }(jQuery));
+    var request = new XMLHttpRequest();
```
Comment (ted-hanson) [inline app/views/render_async/_render_async.html.erb]: I think it would make sense to wrap this in a function to keep the state out of global scope

**Comment by sasharevzin** [comment]
Oh right, I’ll update in the morning.

**Comment by nikolalsvk** [comment]
Looks nice! 👏 

I would just leave out the `console.log` because we don't want to print out stuff in JS console to people who use `render_async` :)

**Comment by sasharevzin** [comment]
@nikolalsvk `console.log` only when there is an issue, might be useful for debugging, no?

**Comment by nikolalsvk** [comment]
It would help with debugging but it's not wise to leave console.log in production code, here's a [stackoverflow question explaining it](https://stackoverflow.com/questions/8002116/should-i-be-removing-console-log-from-production-code).

**Comment by nikolalsvk** [comment]
Thanks @sasharevzin 🌟

**Pull request merged**